script/run_query

```diff
@@ -36,12 +36,22 @@ parser.add_argument(
 )
 parser.add_argument("--dataset_id", help="Destination dataset")
 parser.add_argument("--query_file", help="File path to query to be executed")
+parser.add_argument(
+    "--use_legacy_sql",
+    help="Enable use of legacy SQL; Standard SQL by default",
+    default=False,
+)
 
 
 def main():
     args, query_arguments = parser.parse_known_args()
     query_file = args.query_file
 
+    # bq uses legacy SQL by default, force use of standard SQL if no flag is set
+    query_arguments.append(
+        "--use_legacy_sql={}".format(str(args.use_legacy_sql).lower())
```

Comment:
I believe it only allows `true` or `false` so we need to lowercase the Python bool values

Reply:
I believe it cannot not take an argument. You can specify `--use_legacy_sql` to ensure legacy SQL is used or `--nouse_legacy_sql` to ensure Standard SQL is used.

Reply:
Looks like both `--nouse_legacy_sql` and `--use_legacy_sql=false` is supported. We use the latter here: https://github.com/mozilla/bigquery-etl/blob/107bc37cbd96712b4ee4f92ec06e9b2322e7e69b/.bigqueryrc#L5